src/host/services/IMUService.cpp

```diff
@@ -42,15 +57,26 @@ void IMUService::setup() {
   // Where each can be 00:X, 01: Y, 10:Z
   // Sign is the 3 lsb: 00000xyz
   switch (_config.mounting) {
-    case HorizontalLeftSideToBow:
+    case horizontalTopToBow:
```

Comment:
Do you have time to test and make sure it works well?

If not, can you move all the ones that are untested at the bottom and add a comment to say that they have not been tested yet?